Microsoft.Toolkit.HighPerformance/Extensions/HashCodeExtensions.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if !NETSTANDARD1_4
```

Comment:
Why not `Compile Remove="" Condition="'$(TargetFramework)' != 'netstandard1.4"`?

Reply:
I've tried that but it looks like Visual Studio acts a bit weird when I use that. No matter where I place that `Compile` tag, what condition I specify and what active target I select from the dropdown, the file is just always hidden from the solution explorer. Not really sure why that's happening, to be honest 🤔

Since it's just a couple of files anyway I thought it'd be easier to just disable them with that compiler directive instead of adding more complexity to the `.csproj` file.